src/main/java/dev/aziz/movies/controllers/PeopleController.java

```diff
@@ -0,0 +1,28 @@
+package dev.aziz.movies.controllers;
+
+import dev.aziz.movies.entities.People;
+import dev.aziz.movies.services.PeopleService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class PeopleController {
+
+    private final PeopleService peopleService;
+
+    @GetMapping("/people")
+    public ResponseEntity<List<String>> getAllPeople() {
+        return ResponseEntity.ok(peopleService.getAllPeople());
+    }
+
+    @GetMapping("/people/{person}/{id}")
+    public ResponseEntity<Object> getOnePerson(@PathVariable People person, @PathVariable Long id) {
```

Comment:
Using Object is a very bad practice.

Instead, both ActorDto and DirectorDto have the same fields, why don't create a PersonDto?